doc/nrf/releases/release-notes-latest.rst

```diff
@@ -129,7 +129,40 @@ The current |NCS| release is based on Zephyr v2.5.99.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-* No changes yet
+* Drivers:
+
+  * Flash
+
+    * SoC embedded flash driver enabling imposes selection of
+      :option:`CONFIG_MPU_ALLOW_FLASH_WRITE` if ARM MPU enabled.
+
+
+* Libraries / Subsystems:
+
+  * File systems:
+
+    * Added c:func:`fs_file_t_init` function for initialization of
+      c:type:`fs_file_t` objects. Any :c:type:`fs_file_t` objects must now be
```

Comment:
maybe "must be now (...)" -> "require initialization, by calling the function, prior to their first use."